infra/license/package-blocklist.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
IHMO, simply `package-list.json` could be better.

Reply:
I agree with changing the file name :)
Which one would be better?

- package.json / license.json
- package-list.json / license-list.json
- package-referee.json / license-referee.json
- package-judge.json / license-judge.json
- etc

Reply:
> package-list.json / license-list.json

+1

Reply:
I like package-judge.json and don't like package.json. :-) Others seems fine.

Reply:
(there is word `allow`/`deny` and 'permission')

Reply:
I will use `package-judgment.json` and `license-judgment.json` because
- We **judge** whether we can use package/license or not
- json file includes **judgments** of each package/license
- json file includes both allowance(permission) and denial(forbiddance)

Thanks for good suggestions! :D